### PR TITLE
TRON-39 - account lib skeleton

### PR DESCRIPTION
**Pull request by twtaylorbitgo**
This is the skeleton for account lib. It is mostly based off this document.

https://docs.google.com/document/d/17snNmlElYph59maFdbxF_jCnIF3wDa8e3Wf0fJ5fJdU/edit#heading=h.79rp2kie2cj

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@bitgo/bitgo-account-lib",
+  "version": "1.0.0-rc.0",
```
Comment (argjv) [inline package.json]: This should probably be `0.0.1` and change it to `1.0.0` once we properly add Tron

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,8 @@
+node_modules/
```
Comment (argjv) [inline .gitignore]: Add `.idea/` which is added by IntelliJ

File: package.json (hunk 1 above)
Comment (twtaylorbitgo) [reply package.json]: sure

File: .gitignore (hunk 2 above)
Comment (twtaylorbitgo) [reply .gitignore]: k

**Pull request merged**